recipes/vkmz/meta.yaml

```diff
@@ -16,15 +16,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8
```

Comment:
Does this really need 3.8? It's available, but things haven't been fully migrated to it yet so I fear it will be difficult to install this along side anything else.